resource_booking/models/resource_booking.py

```diff
@@ -0,0 +1,531 @@
+# Copyright 2021 Tecnativa - Jairo Llopis
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+import calendar
+from contextlib import suppress
+from datetime import datetime, timedelta
+
+from dateutil.relativedelta import relativedelta
+
+from odoo import _, api, fields, models
+from odoo.exceptions import ValidationError
+from odoo.osv.expression import NEGATIVE_TERM_OPERATORS
+
+from odoo.addons.resource.models.resource import Intervals
+
+
+class ResourceBooking(models.Model):
+    _name = "resource.booking"
+    _inherit = ["mail.thread", "mail.activity.mixin", "portal.mixin"]
+    _description = "Resource Booking"
+    _order = "start DESC"
+    _sql_constraints = [
+        (
+            "combination_required_if_event",
+            "CHECK(meeting_id IS NULL OR combination_id IS NOT NULL)",
+            "Missing resource booking combination.",
+        ),
+        (
+            "start_stop_together",
+            """CHECK(
+                (start IS NULL AND stop IS NULL) OR
+                (start IS NOT NULL AND stop IS NOT NULL)
+            )""",
+            "Start and stop must be filled or emptied together.",
+        ),
+        (
+            "unique_meeting_id",
+            "UNIQUE(meeting_id)",
+            "Only one event per resource booking can exist.",
+        ),
+    ]
+
+    active = fields.Boolean(index=True, default=True)
```

Comment:
Remove index from boolean field

Reply:
Why? This is used almost always to filter values... 🤔

Reply:
https://stackoverflow.com/a/42972924